datastore/graphers/nmdb-graph-network/nmdb-graph-network.cpp

```diff
@@ -766,6 +775,44 @@ class Tool : public nmdt::AbstractGraphTool
       return label;
     }
 
+
+    std::string
+    getUseIconsCustomPath(std::string customPath)
+    {
+      if (!useIconFolder) {
+        return "";
+      }
+
+      sfs::path imagePath {opts.getValue("icon-folder")};
+      std::string iconPath {imagePath.string()};
+      std::string label {
+        "<TD width=\"60\" height=\"50\" fixedsize=\"true\"><IMG SRC=\""
+      };
+
+
+      if(!customPath.empty()) {
+        for (auto& pathIter : sfs::recursive_directory_iterator(imagePath)) {
+          std::string fileName {pathIter.path().filename()};
+          
+          if (std::equal(customPath.begin(), customPath.end(),
+              fileName.begin(), fileName.end(),
+              [](auto a, auto b)
+              {
+                return std::tolower(a) == std::tolower(b);
+              }))
+          {
+            iconPath = fileName;
+            break;
+          }
+        }
+      }
+
+      label += iconPath + "\" scale=\"true\"/></TD>";
+
+      return label;
+    }
+
+
```

Comment:
This function is not needed and is a copy/paste of the `getUseIconString()` function, save the couple of `imagePath` related lines.

```suggestion
```